src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -599,10 +601,12 @@ class NetworkOPsImp final : public NetworkOPs
 
     Json::Value
     transJson(
-        const STTx& transaction,
+        std::shared_ptr<STTx const> const& transaction,
```

Comment:
That's because this function now calls `insertDeliveredAmount` which takes `std::shared_ptr<STTx const>` ; moving this call here helped to clean the design of this part - basically not other part of this file now touches the transaction JSON output, which IMO is improvement. I did consider changing the signature of `insertDeliveredAmount` to take `const STTx&` instead of `shared_ptr` so I do not need `shared_ptr` here, but realised that would cause large unnecessary churn in other unrelated parts, and that's not a right PR for this kind of change.

Reply:
> That's because this function now calls `insertDeliveredAmount` 

Ah! That makes sense!